actions/plugins/create_release.php

```diff
@@ -84,15 +84,7 @@
 }
 
 $release->setRecommended($recommended);
-
-$release->setHash();

```

Comment:
Hash is now set for all releases during `save()`. It was missing in some actions.